.github/workflows/intergation-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: "Trigger Integration tests"
+on:
+  push:
+    branches:
+      - master
+      - 'releases/*'
+  pull_request:
+    branches:
+      - master
+      - 'releases/*'
+jobs: 
+    trigger-integration-tests:
+      name: Trigger Integration tests
+      runs-on: ubuntu-latest
+      steps:
+        - name: Extract branch name
+          id: extract_branch
+          run: |
+            echo "##[set-output name=branchname;]$(echo ${GITHUB_REF##*/})"
```

Comment:
This is probably not required right?

You could use `${{ env.GITHUB_REF }}` in the condition?

Reply:
This is required as  ${{ env.GITHIB_REF }} gives the full path, the name needs to be extracted